flexget/plugins/modify/convert_magnet.py

```diff
@@ -33,8 +33,10 @@ def magnet_to_torrent(self, magnet_uri, destination_folder, timeout):
         import libtorrent
         params = libtorrent.parse_magnet_uri(magnet_uri)
         session = libtorrent.session()
-        # for some reason the info_hash needs to be bytes but it's a struct called sha1_hash
-        params['info_hash'] = bytes(params['info_hash'])
+        lt_version = [int(v) for v in libtorrent.version.split('.')]
+        if lt_version[0] > 0 or (lt_version[0] == 0 and lt_version[1] == 16 and lt_version[2] > 13):
```

Comment:
Could this be more clearly expressed as `if lt_version > [0, 16, 13]:`? Also, are you sure this is to do with a change since libtorrent 0.16.14, and not something to do with python 2 vs 3? Have, (or can you,) test with both python 2 and 3?